src/main/java/org/micks/champmaker/auth/Controller.java

```diff
@@ -0,0 +1,17 @@
+package org.micks.champmaker.auth;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/auth")
+@CrossOrigin
```

Comment:
@Mickss  have you checked if this endpoint is working without `@CrossOrigin` annotation? Maybe it is not required when calling from other microservices in the same domain.